ui/qt-lib/src/Ariadne/UI/Qt/Widgets/TopBar.hs

```diff
@@ -92,7 +94,7 @@ initTopBar = do
   replButton <- QPushButton.new
   helpButton <- QPushButton.newWithText ("Help" :: String)
   logsButton <- QPushButton.newWithText ("Logs" :: String)
-
+  settingsButton <- QPushButton.newWithText ("Settings" :: String)
```

Comment:
This should be a button with just an icon, like repl button. Take a look at https://zpl.io/VxYE0xX
Text buttons for help and logs dialogs is a temporal solution until those dialogs are redesigned.